src/ocamlorg_data/tutorial.ml

```diff
@@ -18291,7 +18291,7 @@ you can use a producer-consumer arrangement to invert control:</p>
-</span><span class='ocaml-source'>    </span><span class='ocaml-constant-language-capital-identifier'>Event</span><span class='ocaml-keyword-other-ocaml punctuation-other-period punctuation-separator'>.</span><span class='ocaml-source'>sync</span><span class='ocaml-source'> </span><span class='ocaml-source'>(</span><span class='ocaml-constant-language-capital-identifier'>Event</span><span class='ocaml-keyword-other-ocaml punctuation-other-period punctuation-separator'>.</span><span class='ocaml-source'>send</span><span class='ocaml-source'> </span><span class='ocaml-source'>channel</span><span class='ocaml-source'> </span><span class='ocaml-constant-language-capital-identifier'>None</span><span class='ocaml-source'>)</span><span class='ocaml-source'> </span><span class='ocaml-keyword-other'>in</span><span class='ocaml-source'>
+</span><span class='ocaml-source'>    Event</span><span class='ocaml-keyword-other-ocaml punctuation-other-period punctuation-separator'>.</span><span class='ocaml-source'>sync</span><span class='ocaml-source'> </span><span class='ocaml-source'>(</span><span class='ocaml-constant-language-capital-identifier'>Event</span><span class='ocaml-keyword-other-ocaml punctuation-other-period punctuation-separator'>.</span><span class='ocaml-source'>send</span><span class='ocaml-source'> </span><span class='ocaml-source'>channel</span><span class='ocaml-source'> </span><span class='ocaml-constant-language-capital-identifier'>None</span><span class='ocaml-source'>)</span><span class='ocaml-source'> </span><span class='ocaml-keyword-other'>in</span><span class='ocaml-source'>
```

Comment:
I don't think this change is required? Can you elaborate why you made this change @Lontchi12 ?

Reply:
This is very likely not your fault -- this bit of code is generated by things in `ood-gen` so I think it is possibly some combination of differing version of packages / different OSes that caused it. For now lets revert it but I have a feeling it may crop up again :))